### PR TITLE
Bump devDependencies

### PR DESCRIPTION
**Pull request by GretaD**
Signed-off-by: GretaD <gretadoci@gmail.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,22 +73,22 @@
     "vue-tabs-component": "^1.5.0",
     "vuex": "^3.6.2",
     "vuex-router-sync": "^5.0.0",
-    "webdav": "^3.6.2"
+    "webdav": "^4.0.0"
```
Comment (ChristophWurst) [inline package.json:76]: not a dev dependecy

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
why do you update all in one btw?

**Comment by GretaD** [comment]
> why do you update all in one btw?

not a specific reason. I did it because was only one dependency and the rest dev dependency. Should i keep only the devdep?

**Comment by ChristophWurst** [comment]
> not a specific reason. I did it because was only one dependency and the rest dev dependency. Should i keep only the devdep?

let's please do the bump separately if possible so we don't accidentally update anything that could cause breakage. also if we have separate bumps then we can bisect the critical update if there are any regressions later :)

**Comment by GretaD** [comment]
close this and merge the bump individually

**Pull request closed without merging**